importScripts/sql/transformations/ObjectsExportView.sql

```diff
@@ -125,6 +125,28 @@ CREATE OR REPLACE VIEW ObjectsExport AS
         NULLIF(TRIM(o.TreatmentCompleteYesNo), '')                AS TreatmentCompleteYesNo,
         NULLIF(TRIM(o.TreatmentPriority), '')                     AS TreatmentPriority,
         NULLIF(TRIM(o.UsualLocation), '')                         AS UsualLocation,
+        lhc.Room                                                  AS UsualLocationRoom,
+        lhc.Shelf                                                 AS UsualLocationShelf,
+        lhc.Box                                                   AS UsualLocationBox,
+        IF(
+            o.ItemType IN ('Photograph', 'Memorials'),
+            NULL,
+            COALESCE(lhc.Box, lhc.Shelf, lhc.Room, NULLIF(TRIM(o.UsualLocation), ''))
+        )                                                         AS UsualLocationLowest,
+        CASE
+        WHEN lhc.Box IS NOT NULL
+            THEN 'box'
+        WHEN lhc.Shelf IS NOT NULL
+            THEN 'shelf'
+        WHEN lhc.Room IS NOT NULL
+            THEN 'room'
+        WHEN
+            (o.ItemType NOT IN ('Photograph', 'Memorials') AND
+             NULLIF(TRIM(o.UsualLocation), '') IS NOT NULL AND
```

Comment:
This is where I'm defaulting the location value to `UsualLocation` if it's not one of the standardised `LocationHomes`.